contracts/dao/voting/lockdrop-vault/src/msg.rs

```diff
@@ -0,0 +1,35 @@
+use cosmwasm_std::Uint64;
+use neutron_lockdrop_vault::msg::PoolType;
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq, JsonSchema)]
+#[serde(rename_all = "snake_case")]
+pub enum LockdropQueryMsg {
+    QueryUserLockupTotalAtHeight {
+        pool_type: PoolType,
+        user_address: String,
+        height: u64,
+    },
+    QueryLockupTotalAtHeight {
+        pool_type: PoolType,
+        height: u64,
+    },
+}
```

Comment:
Why didn't you import these messages from lockdrop contract and instead copypasted them here?